core/variant/native_ptr.h

```diff
@@ -53,15 +53,23 @@ struct GDExtensionPtr {
 	operator Variant() const { return uint64_t(data); }
 };
 
-#define GDVIRTUAL_NATIVE_PTR(m_type)                                                                                                                                                               \
+#define GDVIRTUAL_NATIVE_PTR(m_type) /***********************************************************************************************************************************************************/ \
```

Comment:
I'm not convinced by this technique.
IMO the resulting code has needlessly long lines, and I prefer we actually reduce the line length now that we can.

![image](https://github.com/user-attachments/assets/2c180c54-38e2-4b87-ba0a-2c41a22c6661)

versus

![image](https://github.com/user-attachments/assets/f9188915-cfc0-4755-8c89-cc91aee8c884)

We have `.git-blame-ignore-revs` to deal with such whitespace changes in the git history.

Reply:
Good point! I'll convert to that then